phpunit

```diff
@@ -10,19 +10,23 @@ require __DIR__.'/src/Symfony/Component/Process/ProcessUtils.php';
 $PHPUNIT_VERSION = PHP_VERSION_ID >= 70000 ? '5.0' : '4.8';
 $PHPUNIT_DIR = __DIR__.'/.phpunit';
 $PHP = defined('PHP_BINARY') ? PHP_BINARY : 'php';
+$PHP = ProcessUtils::escapeArgument($PHP);
+$COMPOSER_INTERPRETER = $PHP;
 
-if (!file_exists($COMPOSER = __DIR__.'/composer.phar')) {
+if (!file_exists($COMPOSER = __DIR__ . '/composer.phar')) {
```

Comment:
@sstok Ooop. Sorry about that. That was my editor auto-fixing styling. :) Reverted.